### DISCUSSION TITLE
Backwards compatibility

### DISCUSSION THREAD

**Discussion by rodrigargar** [Altinity/clickhouse-backup#266] [Q&A] [answered]
Is clickhouse-backup 1.0.0 able to restore backups created with version 0.6.4?

In my tests, I get an _is empty backup_ error when I try to download with version 1.0.0 a non-empty backup created with version 0.6.4:
`[root@ch01-shard02-replica03 ~]# clickhouse-backup download 2021-09-14T09:00:00-full-shard2-0.6.4`
`2021/09/15 10:17:24 error '2021-09-14T09:00:00-full-shard2-0.6.4' is empty backup`

The same backup is correctly downloaded and restored by version 0.6.4.

**Comment by Slach** [comment, marked as answer]
old format backup is a one monolith archive
look like we have bug when try detect empty backup before try to download legacy backup data|

`ALLOW_EMPTY_BACKUPS=1 clickhouse-backup download <backup-name>`
as quick workaround
